src/Components/folder/FolderOption.jsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { useFolder } from "context/FolderContext";
+import { useModal } from "hooks/useModal";
+import ModalContainer from "../common/Modal/Modal";
+import * as Modal from "../common/Modal/Modal";
+import SocialIcons from "../common/Modal/SocialIcons";
+
+const OptionList = styled.div`
+  display: flex;
+  gap: 12px;
+
+  @media (max-width: 767px) {
+    margin-top: 12px;
+  }
+`;
+
+const Option = styled.div`
+  display: flex;
+  gap: 4px;
+  color: var(--Linkbrary-gray60, #9fa6b2);
+  font-size: 14px;
+  font-weight: 600;
+  cursor: pointer;
+`;
+
+function FolderOption() {
+  const { folderFilter } = useFolder();
+  const { openModal, handleModalOpen, handleModalClose } = useModal();
+  const [option, setOption] = useState("");
+
+  return folderFilter.name === "전체" ? (
+    ""
+  ) : (
```

Comment:
@78-artilleryman 

```suggestion
  return folderFilter.name === "전체" ? (
    ""
  ) : (
```

이 부분도 early return을 통해 불필요한 ternary operator를 렌더링 로직에 포함시킬 필요는 없어보여요!